test/test_core.py

```diff
@@ -6528,14 +6532,18 @@ def test_sse1(self):
   @is_slow_test
   @no_ubsan('https://github.com/emscripten-core/emscripten/issues/19688')
   @no_asan('local count too large')
-  def test_sse2(self):
+  @parameterized({
+    '': ([],),
```

Comment:
let's make this an explicit `-mno-nontrapping-fptoint' so that when we flip the default, it will still be testing the same case.

Reply:
I think maybe better to explicitly change these tests when we make that change, since the variant will want to be renamed so the default is still empty string and the non-default becomes perhaps `-trapping`?  Or `-no-nontrapping`?

Reply:
or we could just name both variants I guess.

Reply:
I'm kind keen on the idea of always having an unnamed variant for the default.  Then there is always a test you can run that matches the name of the test in python.  i.e. if I see `def test_foo_bar` i know I can run a test called `test_foo_bar` without any suffix.   Maybe no a necessarily things, but it seems nice to have.

Reply:
We already have other tests here like `test_fasta_nontrapping` that will need to be inverted when we change the default.